.ci/deps.pip.sh

```diff
@@ -13,7 +13,7 @@ esac
 
 for dep_version in "${dep_versions[@]}" ; do
   pyenv install -ks $dep_version
-  pyenv local $dep_version 2.7.10
+  pyenv local $dep_version #2.7.10
```

Comment:
err, it cant get merged like that.

Reply:
Sorry, I was just testing and on using python image for circleci2.0,
Python version 2.7.10 was not installed itself...
So, I just commented the version